src/custom/state/application/initialState.ts

```diff
@@ -0,0 +1,15 @@
+import { ApplicationState } from '@src/state/application/reducer'
+import { localWarning } from 'components/Header/URLWarning/localWarning'
+
+const popupList: ApplicationState['popupList'] = []
+
+if (localWarning) {
+  popupList.push({
+    key: 'missingPinataEnvVars',
```

Comment:
we can give a more generic name instead of assuming this is for pinata.

`warningMessage` should be fine